python/setup_cluster/setup_nomad.py

```diff
@@ -0,0 +1,160 @@
+from setup_cluster.ssh_client_handler import SSHClientHandler
```

Comment:
This should be done in a different PR, but just commenting again here since it has to do with this file. Our nomad setup needs to be refactored a bit. 
We can use systemctl to handle the nomad clients/server instead of making tmux sessions.
Also, currently we clone our github repo and use some files there. I don't think that is necessary, we could probably just process some files locally, scp those to the nodes, and use only what's needed.